godtools/Managers/LanguagesManager.swift

```diff
@@ -75,16 +75,18 @@ class LanguagesManager: GTDataManager {
         if GTSettings.shared.primaryLanguageId == nil {
             return nil
         }
-        
-        return loadFromDisk(id: GTSettings.shared.primaryLanguageId!)
+        let primaryLanguage: Language? = loadFromDisk(id: GTSettings.shared.primaryLanguageId!)
+        UserDefaults.standard.set((primaryLanguage?.code ?? ""), forKey: "kPrimaryLanguageCode")
```

Comment:
I think there's a better place to set the primary code, instead of when loading the primary language. Can we set the code when the ID gets set during language selection? That seems more consistent.

Reply:
I moved setting these values to new spots. We can get the parallel language code when they select from the tableview, but to get primary code, I had to move it to the defaultLanguage getter and setter.  Suggestions for alternatives?